### DISCUSSION TITLE
Loose empty translation

### DISCUSSION THREAD

**Discussion by nikeddarn** [spatie/laravel-translatable#228] [Q&A]
When multiply properties are empty string like that

'slug' => [
         'en' => '',
         'ar' => '',
    ]

we have only one property in json string, like:

{"ar": ""}

I override getTranslations method in HasTranslations.php and it works.

**Comment by lucassimines** [comment]
I'm having the same problem!

**Comment by serdud** [comment]
+1

**Comment by kolotovio** [comment]
+1 🤷‍♂️

**Comment by mindaugasva** [comment]
still the same problem, can you fix it?
